src/backend/controllers/HistoryController.js

```diff
@@ -54,16 +54,20 @@ export const addVideoToHistoryHandler = function (schema, request) {
       );
     }
     const { video } = JSON.parse(request.requestBody);
-    if (user.history.some((item) => item.id === video.id)) {
-      return new Response(
-        409,
-        {},
-        {
-          errors: ["The video is already in your history"],
-        }
-      );
+    if (user.history.some((item) => item._id === video._id)) {
+      user.history = user.history.filter((item) => item._id !== video._id);
+      user.history.unshift(video);
+
+      // return new Response(
+      //   409,
+      //   {},
+      //   {
+      //     errors: ["The video is already in your history"],
```

Comment:
You can remove comments code

Reply:
Thankyou for your comment 
Will remove this in the next pr